src/components/VCheckbox.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <label :for="id" class="checkbox-label" :class="labelClasses">
+    <input
+      :id="id"
+      type="checkbox"
+      class="checkbox"
+      v-bind="inputAttrs"
+      @change="onChange"
+    />
+    <Checkmark
+      v-show="localCheckedState"
+      class="checkmark"
+      focusable="false"
+      width="20"
+      height="20"
+      role="img"
+    />
+    <!--  @slot The checkbox label  --><slot />
+  </label>
+</template>
+
+<script>
+import Checkmark from '~/assets/icons/checkmark.svg?inline'
```

Comment:
Nice, I didn't know this was possible! (Might be good to standardise usage between inline SVG imports and the `VIcon` component.)

Reply:
It's the [`vue-svg-loader` that is used by the Nuxt `svg-module`](https://github.com/nuxt-community/svg-module#vue-svg-loader). The sad thing about it is I don't think they support Nuxt 3...